src/std/text/json/util.ss

```diff
@@ -134,9 +135,65 @@
 (defclass JSON ())
 (defmethod {:json JSON} trivial-class->json-object)
 
-(def (pretty-json object (out #f))
+(def (pretty-json object (out #f)
```

Comment:
this only supports ports, no go.

Please use a (buffered?) writer to support the io zoo.